src/components/FigureRow.js

```diff
@@ -1,21 +1,47 @@
-import React, { useState } from 'react';
+import React from 'react';
 import Row from 'react-bootstrap/Row';
 import Figure from 'react-bootstrap/Figure';
 import FigureImage from 'react-bootstrap/FigureImage';
-import FigureCaption from 'react-bootstrap/FigureCaption';
 
-function FigureRow({plotvar, evalagainst, regmodel, timeperiod}){
+function FigureRow({plotvars, seasons, periods, plottypes}){
+/*
+  This displays the plots after checking the options chosen by the user.
+  It will display something as soon as there's a choice made for each of the 4
+  drowdowns. It will also automatically delete images as user deselects a
+  choice.
+*/
 
-  return(
-    <Row className="py-4">
-       <Figure>
-         <Figure.Image
-           alt="Image NOT found"
-           src={`${process.env.PUBLIC_URL}/images/rmse-decomposition_${plotvar.value}_${evalagainst.value}_${regmodel.value}_${timeperiod.value}.png`}
-         />
-       </Figure>
-     </Row>
-  );
+  // Don't try to show a plot until the user has selected at least one value of
+  // each dropdown
+  if ([plotvars, seasons, periods, plottypes].some(ar => ar.length === 0)) {
+    console.log("error, you didnt make enough selections");
+    return(null);
+  }
+  else {
+    // Once selection is made, group plots by plot type, then delve into the
+    // other selected options.
+    // TODO: check with research team this is the best way to proceed.
+    return(
+      <>
+        {plotvars.map(plotvar => (
+          seasons.map(season => (
+            periods.map(period => (
+              plottypes.map(plottype => (
+                <Row className="py-4">
+                  <Figure>
+                    <FigureImage
+                     alt="Image NOT found"
+                     src={`${process.env.PUBLIC_URL}/images/${plotvar.value}/${plottype.value}_${plotvar.value}_${season.value}_${period.value}.png`}
+                    />
+                  </Figure>
```

Comment:
I used to have a FigureCaption, but the plots have a title on them so it was duplicating information. I think in later versions Clair will remove that title, so I'll bring the FigureCaption back.

Is that what you meant?

Reply:
When the image is not found, though, it'd help indeed knowing which image is the one that wasn't found.

Reply:
There is a title, but I don't think it includes season and period so it wasn't clear which plot was which if I selected multiple seasons.

Reply:
Ah, right. The season is referred to with an identifier in the titles, so Summer is "JJA" (June, July and August), for example.
Regarding the decade, you are right! I think titles will have to be reworked as I think depending on how we decide to group the plots we might have those titles added in React rather than inside @clairbarnes' plots.

Reply:
Created #24 so we don't forget to discuss this.